services/venice-controller/src/main/java/com/linkedin/venice/controller/InstanceRemovableStatuses.java

```diff
@@ -0,0 +1,39 @@
+package com.linkedin.venice.controller;
+
+import java.util.List;
+import java.util.Map;
+
+
+public class InstanceRemovableStatuses {
+  private List<String> stoppableInstances;
+  private Map<String, String> nonStoppableInstancesWithReasons;
+  private String redirectUrl;
+
+  public void setRedirectUrl(String redirectUrl) {
+    this.redirectUrl = redirectUrl;
+  }
+
+  public String getRedirectUrl() {
+    return redirectUrl;
+  }
```

Comment:
Let's add `@JsonIgnore` annotation for these two as we don't want them to be a part of the json output